packages/create-next-app/helpers/copy.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { async as glob } from 'fast-glob'
+import path from 'path'
+import fs from 'fs'
+
+interface CopyOption {
+  cwd?: string
+  rename?: (basename: string) => string
+  parents?: boolean
+}
+
+const identity = (x: string) => x
+
+export const copy = async (
+  src: string | string[],
+  dest: string,
+  { cwd, rename = identity, parents = true }: CopyOption = {}
+) => {
+  const source = typeof src === 'string' ? [src] : src
+
+  if (source.length === 0 || !dest) {
+    throw new TypeError('`files` and `destination` are required')
+  }
+
+  const sourceFiles = await glob<string>(source, {
+    cwd,
+    dot: true,
+    absolute: false,
+    stats: false,
+  })
+
+  const destRelativeToCwd = cwd ? path.resolve(cwd, dest) : dest
+
+  return Promise.all(
+    sourceFiles.map(async (p) => {
+      const dirname = path.dirname(p)
+      const basename = rename(path.basename(p))
+
+      const from = cwd ? path.resolve(cwd, p) : p
+      const to = parents
+        ? path.join(destRelativeToCwd, dirname, basename)
+        : path.join(destRelativeToCwd, basename)
+
+      // This ensure the destination directory exists
```

Comment:
```suggestion
      // Ensure the destination directory exists
```

Reply:
Fixed in [`04dcc3b` (#53241)](https://github.com/vercel/next.js/pull/53241/commits/04dcc3b773e54e08be5a86974aa204909a67cb0f)